### DISCUSSION TITLE
`panels::auth.password-reset` render hooks

### DISCUSSION THREAD

**Discussion by cawecoy** [filamentphp/filament#10506] [Ideas]
Filament doesn't have render hooks for some auth pages that I need to customize:

- `panels::auth.password-reset.request.form.before`
- `panels::auth.password-reset.reset.form.before`

**Comment by devhoussam1998** [comment]
you can add a PR

**Reply by cawecoy** [reply]
PR added: https://github.com/filamentphp/filament/pull/10752
